configuration/cdap-site.xml

```diff
@@ -1138,117 +1184,350 @@
     <value-attributes>
       <type>int</type>
-      <minimum>0</minimum>
-      <maximum>268435456</maximum>
-      <unit>MB</unit>
-      <increment-step>256</increment-step>
+      <minimum>1</minimum>
       <overridable>false</overridable>
     </value-attributes>
   </property>
 
   <property>
-    <name>metrics.dataset.hbase.stats.report.interval</name>
-    <value>60</value>
-    <description>Report interval for HBase stats, in seconds</description>
+    <name>data.queue.dequeue.tx.percent</name>
+    <value>30</value>
+    <description>
+      Percentage of transaction time allowed to spend in dequeue; it should
+      be an integer between 1-100
+    </description>
+    <value-attributes>
+      <type>int</type>
+      <minimum>1</minimum>
+      <maximum>100</maximum>
+      <overridable>false</overridable>
+    </value-attributes>
   </property>
 
-  <!-- Logging Configuration -->
+  <property>
+    <name>data.queue.table.presplits</name>
+    <value>16</value>
+    <description>
+      Number of splits in the queue table
+    </description>
+    <value-attributes>
+      <type>int</type>
+      <minimum>1</minimum>
+      <overridable>false</overridable>
+    </value-attributes>
+  </property>
+
+  <!-- Router Configuration -->
 
   <property>
-    <name>kafka.seed.brokers</name>
-    <value>{{cdap_kafka_brokers}}</value>
+    <name>router.bind.address</name>
+    <value>0.0.0.0</value>
+    <description>
+      CDAP Router service bind address
+    </description>
+  </property>
+
+  <property>
+    <name>router.bind.port</name>
+    <value>11015</value>
+    <description>
+      CDAP Router service bind port
+    </description>
+  </property>
+
+  <property>
+    <name>router.client.boss.threads</name>
+    <value>1</value>
+    <description>The number of boss threads in the CDAP Router service client</description>
+    <value-attributes>
+      <type>int</type>
+      <minimum>1</minimum>
+      <overridable>false</overridable>
+    </value-attributes>
+  </property>
+
+  <property>
+    <name>router.client.worker.threads</name>
+    <value>10</value>
+    <description>The number of worker threads in the CDAP Router service client</description>
+    <value-attributes>
+      <type>int</type>
+      <minimum>1</minimum>
+      <overridable>false</overridable>
+    </value-attributes>
+  </property>
+
+  <property>
+    <name>router.connection.backlog</name>
+    <value>20000</value>
+    <description>The connection backlog in the CDAP Router service</description>
+  </property>
+
+  <property>
+    <name>router.connection.idle.timeout.secs</name>
+    <value>15</value>
+    <description>
+      The number of seconds after an HTTP request completes that idle router connections are closed
+    </description>
+  </property>
+
+  <property>
+    <name>router.server.address</name>
+    <value>{{cdap_router_host}}</value>
+    <description>
+      CDAP Router service address to which CDAP UI connects
+    </description>
+  </property>
+
```

Comment:
Thanks! Somehow I missed that. I kept it commented at the bottom of the file and had asked in Slack if it had been removed. Didn't get a response. I'll restore it.